analysis/tests/src/CompletionFromModule.res

```diff
@@ -0,0 +1,39 @@
+module SomeModule = {
+  type t = {name: string}
+
+  @get external getName: t => string = "name"
+
+  let thisShouldNotBeCompletedFor = () => "hi"
+}
+
+let n = {SomeModule.name: "hello"}
+
+// ^dv+
+// n.
+//   ^com
+// ^dv-
+
+@mainTypeForModule(SomeOtherModule)
```

Comment:
Would this also work when `SomeOtherModule` is another file?

Reply:
Yes, it's effectively just a path to a module, so it'll take that exact module path and try to complete it from whatever point you're completing. So it's simulating doing `whateverToComplete-><pathFromMainTypeForModuleHere>.<com>`.